lang/en/subcourse.php

```diff
@@ -71,6 +71,8 @@
 $string['settings:coursepageprintgrade_desc'] = 'Display grade from referenced course on course page.';
 $string['settings:coursepageprintprogress'] = 'Progress on course page';
 $string['settings:coursepageprintprogress_desc'] = 'Display progress from referenced course on course page.';
+$string['settings:displayhiddencourses'] = 'Display Hidden Courses';
+$string['settings:displayhiddencourses_desc'] = 'When selecting a subcourse from list hidden coruses will be displayed.';
```

Comment:
typo "coruses" -> "courses" but some better text might be something like:
"Allow hidden courses to be selected when editing a subcourse"